framework/global/types/uri.cpp

```
@@ -98,6 +98,31 @@ void Uri::setPath(const std::string& path)
     m_path = path;
 }
 
+void Uri::addPath(const std::string& path)
```

Cppcheck (2.21.0)

[style] The function 'addPath' is never used.

(unusedFunction)